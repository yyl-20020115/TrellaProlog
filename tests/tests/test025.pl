:-initialization(main).

main :- findall(integer(I),between(1,10,I),L), write(L), nl.
