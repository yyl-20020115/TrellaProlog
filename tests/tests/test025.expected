[integer(1),integer(2),integer(3),integer(4),integer(5),integer(6),integer(7),integer(8),integer(9),integer(10)]
